tests/vm_test.go

```diff
@@ -2109,3 +2110,37 @@ func stopVM(virtClient kubecli.KubevirtClient, vm *v1.VirtualMachine) *v1.Virtua
 
 	return vm
 }
+
+func newRandomVirtualMachineInstanceWithFileDisk(namespace string, accessMode k8sv1.PersistentVolumeAccessMode) (*v1.VirtualMachineInstance, *cdiv1.DataVolume) {
+	sc, foundSC := libstorage.GetRWOFileSystemStorageClass()
+	if accessMode == k8sv1.ReadWriteMany {
+		sc, foundSC = libstorage.GetRWXFileSystemStorageClass()
+	}
+	if !foundSC {
+		Fail("Skip test when Filesystem storage is not present")
+	}
+
+	virtCli := kubevirt.Client()
+
+	dv := libdv.NewDataVolume(
+		libdv.WithRegistryURLSourceAndPullMethod(cd.DataVolumeImportUrlForContainerDisk(cd.ContainerDiskAlpine), cdiv1.RegistryPullNode),
+		libdv.WithPVC(
+			libdv.PVCWithStorageClass(sc),
+			libdv.PVCWithVolumeSize(cd.ContainerDiskSizeBySourceURL(cd.DataVolumeImportUrlForContainerDisk(cd.ContainerDiskAlpine))),
+			libdv.PVCWithAccessMode(accessMode),
+			libdv.PVCWithVolumeMode(k8sv1.PersistentVolumeFilesystem),
+		),
+	)
+
+	var err error
+	dv, err = virtCli.CdiClient().CdiV1beta1().DataVolumes(namespace).Create(context.Background(), dv, metav1.CreateOptions{})
+	Expect(err).ToNot(HaveOccurred())
+	libstorage.EventuallyDV(dv, 240, Or(HaveSucceeded(), WaitForFirstConsumer()))
+	return libvmi.New(
+		libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
+		libvmi.WithNetwork(v1.DefaultPodNetwork()),
+		libvmi.WithDataVolume("disk0", dv.Name),
+		libvmi.WithResourceMemory("1Gi"),
```

Comment:
Why?

Reply:
It is used in the original code: https://github.com/kubevirt/kubevirt/blob/7f8ebfa0984951b8e52ea148aca006768ea2b732/tests/utils.go#L195

Reply:
@xpivarc The aim of the PR as the title suggests is to move a certain function from `utils.go`, by your question do you suggest to improve the VM spec in order to reduce CI resources? Do you suggest to use a built-in VMs from libvmi?

Reply:
@avlitman while we visit these functions we should also thing about the correctness. There is no need to have the `1Gi`, please use the same as the factory does.

@enp0s3 
The title is lie, talking about functions but only one function is moved. I would even argue that it is not moving a function but rather replacing it. I would even stretch it and say that this function is still needed to some point as it is reoccurring logic. All I am asking is to think about these things when we do find the time to work on improvements/removals in the test suite because moving files has little to no value in its self. Also it reduces the CI resources if we do more in one PR than simple move.